### ISSUE TITLE
gh-pages: page menu doesn't work

### ISSUE THREAD

**Issue by JoshAddington** [rosarior/awesome-django#120]
When using the main Table of Contents, links do not correctly link to headings throughout the page. This is because the links reference #heading-title while the headings have a class="awesome-django-heading-title"

 The sidebar menu works fine, as the link reference these is #awesome-django-heading-title.

**Comment by rosarior** [comment]
Yeah, I'm using a tool that generates the site from the README.md (http://ricostacruz.com/flatdoc/), guess I'll have to check the docs or code to see why this is happening.

**Comment by rosarior** [comment]
I'm open to include something else like Jekyll or Hyde to have more control about how links are generated. This would allow more control other the theme/template look too.
